packages/installations/src/functions/get-token.ts

```diff
@@ -23,14 +23,17 @@ import { AppConfig } from '../interfaces/app-config';
 import { RequestStatus } from '../interfaces/installation-entry';
 import { ERROR_FACTORY, ErrorCode } from '../util/errors';
 
-export async function getToken(app: FirebaseApp): Promise<string> {
+export async function getToken(
+  app: FirebaseApp,
+  forceRefresh?: boolean
+): Promise<string> {
   const appConfig = extractAppConfig(app);
 
   await completeInstallationRegistration(appConfig);
 
   // At this point we either have a Registered Installation in the DB, or we've
   // already thrown an error.
-  return refreshAuthToken(appConfig);
+  return refreshAuthToken(appConfig, forceRefresh);
```

Comment:
If you don't pass forceRefresh to getToken, this'll be undefined, right? Tbh I'd rather be explicit, make it `= false` here and pass the boolean down? I didn't even realise that that worked with an explicit undefined parameter :/

Reply:
Passing `undefined` explicitly or not passing anything work exactly the same in JS, so I decided to keep the default value definition in the actual function that uses the value, but if it does look that scary, I can change it. :smile: